DESCRIPTION

```diff
@@ -1,16 +1,18 @@
 Package: PKPDsim
 Type: Package
 Title: Simulate ODE models
-Version: 1.0.7
-Date: 2018-06-13
+Version: 1.0.124
+Date: 2021-06-08
 Author: Ron Keizer
-Maintainer: Ron Keizer <ronkeizer@gmail.com>
+Maintainer: Ron Keizer <ron@insight-rx.com>
 Depends: R (>= 3.0.2)
-Imports: Rcpp (>= 0.12.9), BH, dplyr, data.table, lubridate, stringr,
-        MASS, randtoolbox, jsonlite, stats, httr
+Imports: Rcpp (>= 0.12.9), BH, data.table, lubridate, stringr, MASS,
+        randtoolbox, jsonlite, stats, parallel
+Suggests: httr, testit, testthat (>= 3.0.0)
```

Comment:
do we still need testit? (can be a separate pr)

Reply:
I think you're right, we don't need it. Will remove.